rclpy/src/rclpy/_rclpy_logging.c

```diff
@@ -0,0 +1,129 @@
+// Copyright 2017 Open Source Robotics Foundation, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <Python.h>
+
+#include <rcutils/logging.h>
+#include <rcutils/logging_macros.h>
+#include <rcutils/time.h>
+
+/// Initialize the logging system.
+/**
+ * \return None
+ */
+static PyObject *
+rclpy_logging_initialize()
+{
+  rcutils_logging_initialize();
+  Py_RETURN_NONE;
+}
```

Comment:
A situation would be that the user wants to register their own output handler or do some other configuration that should take place after initialisation but before logging. Configuration support isn't there at the moment and the initialize function isn't being used in this PR anymore (since https://github.com/ros2/rcutils/pull/52), but I still left it around since it'll be useful eventually. It might be better practice to not add it until it's needed: I can go either way